be/test/vec/core/get_common_type_test.cpp

```diff
@@ -143,10 +146,8 @@ INSTANTIATE_TEST_SUITE_P(data_type, LeastSuperTypeTest,
                                  {"Float32 Float64", "Float64"},
                                  {"Float32 UInt16 Int32", "Float64"},
                                  {"Float32 Int16 UInt32", "Float64"},
-                                 {"Date Date", "Date"},
-                                 {"DateTime DateTime", "DateTime"},
                                  {"String String String", "String"},
-                                 {"Int8 String", nullptr},
```

Comment:
I think could remove those, no need change to Jsonb

Reply:
need to change to jsonb， since they conflict

Reply:
could remove those test case of return nullptr
now return jsonb seems the default result as get_least_supertype_jsonb.
so those test case is useless

Reply:
done